### ISSUE TITLE
make a11y work in the CI and jest cli

### ISSUE THREAD

**Issue by shairez** [qwikifiers/qwik-ui#308]
currently the a11y tests run when storybook is opened, but not on CI

**Comment by kjugi** [comment]
It is possible to run i.e. py11y-ci but that would run on the `docs/headless/*` and `docs/tailwind/*`pages after generating a sitemap. Ideally, the test should check only the component instead whole page. In that case, I wonder if it's possible.

Also, what accessibility standard should be taken as default?

In jest cli case - I can try to add https://github.com/nickcolley/jest-axe if that suits the lib needs

**Comment by shairez** [comment]
thanks @kjugi !

We tried to run the storybook interaction tests using `test-storybook` but we ran into some nasty issues with jest + TS (+ vite stuff).

So  we decided to try cypress for component testing, which can also handle the a11y testing. 

The progress is in #323 
DM me on discord if you want to join that effort

Thanks again!

**Comment by shairez** [comment]
still need to connect it to CI

**Comment by shairez** [comment]
fixed by #328
